diff --git a/node_modules/ffjavascript/build/browser.esm.js b/node_modules/ffjavascript/build/browser.esm.js
index 439dcfd..514722c 100644
--- a/node_modules/ffjavascript/build/browser.esm.js
+++ b/node_modules/ffjavascript/build/browser.esm.js
@@ -488,7 +488,7 @@ class PolField {
         return v;
     }
 
-    eval(p,x) {
+    evalFix(p,x) {
         const F = this.F;
         if (p.length == 0) return F.zero;
         const m = this._next2Power(p.length);
@@ -519,7 +519,7 @@ class PolField {
             let mpol = this.ruffini(roots, points[i][0]);
             const factor =
                 this.F.mul(
-                    this.F.inv(this.eval(mpol, points[i][0])),
+                    this.F.inv(this.evalFix(mpol, points[i][0])),
                     points[i][1]);
             mpol = this.mulScalar(mpol, factor);
             sum = this.add(sum, mpol);
@@ -15661,7 +15661,7 @@ function thread(self) {
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
+var Worker = {};
 var browser = Worker;
 
 var Worker$1 = /*@__PURE__*/getDefaultExportFromCjs(browser);
